paig-server/backend/requirements.txt

```diff
@@ -33,4 +33,5 @@ httpx==0.27.0
 fasteners
 APScheduler==3.10.4
 pydantic[email]
-posthog
\ No newline at end of file
+posthog
+paig-authorizer-core==0.0.1
```

Comment:
can we keep version open like paig_common

Reply:
I understand the suggestion to specify the dependency as paig-authorizer-core without a version number. However, I believe it's important to include the exact version (e.g., paig-authorizer-core==0.0.1) to ensure compatibility and avoid any potential issues that may arise from version changes in the future.

For example, if a user installs paig-server 0.0.2, and if we don't specify any specific version of paig-authorizer-core, they might inadvertently receive a newer version of the library that includes breaking changes or altered functionality and might not work with paig-server 0.0.2. This could lead to incompatibility issues where the behavior of my project does not work as intended with the latest version. By specifying the version, I can help ensure that users have the correct version that was tested and verified to work with my project, providing a smoother experience.